clang/lib/Analysis/Scalable/Model/EntityIdTable.cpp

```diff
@@ -0,0 +1,40 @@
+//===- EntityIdTable.cpp ----------------------------------------*- C++ -*-===//
```

Comment:
I've seen that in other cpp files you don't have `-*- C++ -*-` part. Strictly speaking, in cpp files the editor already knows that it's C++, so this only really applies to header files.
I don't mind this either way, but keep it consistent.
Probably, we should just have this special comment for the headers where they were designed to apply.